cmake/roslint-extras.cmake.em

```diff
@@ -62,14 +62,14 @@ function(roslint_python)
     file(GLOB_RECURSE ARGN *.py)
   endif()
   if (NOT DEFINED ROSLINT_PYTHON_CMD)
-    set(ROSLINT_PYTHON_CMD ${ROSLINT_SCRIPTS_DIR}/pycodestyle)
+    set(ROSLINT_PYTHON_CMD ${PYTHON_EXECUTABLE} -m roslint.pycodestyle_wrapper)
   endif()
   roslint_custom("${ROSLINT_PYTHON_CMD}" "${ROSLINT_PYTHON_OPTS}" ${ARGN})
 endfunction()
 
 # Run roslint for this package as a test.
 function(roslint_add_test)
   catkin_run_tests_target("roslint" "package" "roslint-${PROJECT_NAME}.xml"
-    COMMAND ${ROSLINT_SCRIPTS_DIR}/test_wrapper ${CATKIN_TEST_RESULTS_DIR}/${PROJECT_NAME}/roslint-${PROJECT_NAME}.xml make roslint_${PROJECT_NAME}
+    COMMAND "${ROSLINT_SCRIPTS_DIR}/test_wrapper ${CATKIN_TEST_RESULTS_DIR}/${PROJECT_NAME}/roslint-${PROJECT_NAME}.xml make roslint_${PROJECT_NAME}"
```

Comment:
These quotes are the fix to the issue that was messing with my testing before; I think there must have been a change in CMake behaviour triggered by #77.